frontend/template-partials/views/accessibility.html

```diff
@@ -0,0 +1,55 @@
+{{<layout}}
+
+  {{$propositionHeader}}{{/propositionHeader}}
+
+  {{$header}}
+    {{header}}
```

Comment:
should this be `{{#t}}accessibility-header{{/t}}`? or something like that

Reply:
No this works fine. But the title of the page is quite long. So going to make it 
```
{{#t}}base.accessibility{{/t}}
```
instead which is shorter otherwise the information in the tab is very long.

Reply:
Actually this works fine.